### PR TITLE
Create feedback link AB#76661

### PR DESCRIPTION
**Pull request by HarryAmmon**
#### Fixes #0000
Adds a mailto link on the word 'feedback', allows for both email address and subject to be controlled via props
#### Checklist

- [ ] Includes tests
- [X] Update documentation

### CODE REVIEW

**Review by sussexrick — COMMENTED** [review]
Good start, but the text and the email address should be fed in as props. Ultimately these should come out of Sitecore.

File: packages/layout/src/components/beta/beta.tsx [hunk 1]
```diff
@@ -1,15 +1,12 @@
...
 type BetaHeaderProps = {
-	text: any;
+	text?: string;
 };
-export const BetaHeader: React.FC<BetaHeaderProps> = ({ text }) => {
-	const TextComponent = useMemo(() => {
-		return typeof text === 'function' ? text : () => text;
-	}, [text]);
 
+export const BetaHeader: React.FC<BetaHeaderProps> = ({ text }) => {
```
Comment (Sitecorgi) [inline packages/layout/src/components/beta/beta.tsx:9]: Definitely want to keep using either useMemo or React.Memo higher order component here for performance.

Comment (HarryAmmon) [reply packages/layout/src/components/beta/beta.tsx]: Reverted to using useMemo with HOC

File: packages/layout/src/components/beta/beta.tsx [hunk 2]
```diff
@@ -1,14 +1,38 @@
...
+							anchorTag={true}
+							href={
+								mail
+									? `mailto:${mail.email || ''}&subject=${mail.subject || ''}`
+									: 'mailto:'
+							}
+						>
+							feedback
+						</Link>{' '}
+						will help us improve
```
Comment (Sitecorgi) [inline packages/layout/src/components/beta/beta.tsx:32]: improve it?

**Review by Sitecorgi — APPROVED** [review]

**Pull request merged**